features/createroom/impl/src/main/kotlin/io/element/android/features/createroom/impl/configureroom/ConfigureRoomPresenter.kt

```diff
@@ -175,6 +176,7 @@ class ConfigureRoomPresenter @Inject constructor(
                     isEncrypted = config.roomVisibility is RoomVisibilityState.Private,
                     isDirect = false,
                     visibility = RoomVisibility.Private,
+                    historyVisibilityOverride = RoomHistoryVisibility.Invited,
```

Comment:
We may want to check that in a unit test, but it's more work. `givenCreateRoomResult` is not enough, we should use a lambdaRecorder. Can be done later maybe.